bb-aem-integration-tests/src/test/java/com/cognifide/bdd/demo/aem/touchui/SiteadminTest.java

```diff
@@ -144,7 +144,7 @@ public void shouldMovePageProperly() {
   private void createPage(String testedPage) {
     siteadminPage.open(CONTEXT_PATH)
         .createNewPage(testedPage, testedPage.toLowerCase(), "Mobile Product Page");
-    siteadminPage.open(CONTEXT_PATH);
```

Comment:
sometimes the newly created page was not listed in siteadmin grid which was causing an error